libflux/go/libflux/parser.go

```diff
@@ -94,12 +94,17 @@ func (p *ASTPkg) String() string {
 	return fmt.Sprintf("%p", p.ptr)
 }
 
-// Parse will take a string and return a parsed source file.
-func Parse(s string) *ASTPkg {
-	cstr := C.CString(s)
-	defer C.free(unsafe.Pointer(cstr))
+func ParseString(src string) *ASTPkg {
+	return Parse("", src)
```

Comment:
There are places where we just pass in some Flux strings, not files, I think it makes it easier just call this rather than specifying empty strings for all those calls. Agree?

Reply:
Sure, makes sense.